manifests/machineconfigdaemon/clusterrole.yaml

```diff
@@ -10,6 +10,12 @@ rules:
 - apiGroups: [""]
   resources: ["pods"]
   verbs: ["*"]
+- apiGroups: ["extensions"]
```

Comment:
Most people use apps/v1 Daemonsets. Daemonsets were graduated from extensions to apps.

Reply:
Looks like the drain lib uses old client: https://github.com/openshift/machine-config-operator/blob/master/vendor/github.com/openshift/kubernetes-drain/drain.go#L41

we can use `extensions` for now, can you add a TODO / open a issue to switch it to `apps` after we have updated the library.

Reply:
https://github.com/openshift/machine-config-operator/issues/71